pkg/workloads/console/runner/runner.go

```diff
@@ -128,7 +309,32 @@ func (c *Runner) FindConsoleByName(namespace, name string) (*workloadsv1alpha1.C
 	return &matchingConsoles[0], nil
 }
 
-func (c *Runner) ListConsolesByLabelsAndUser(namespace, username, labelSelector string) ([]workloadsv1alpha1.Console, error) {
+type ConsoleSlice []workloadsv1alpha1.Console
+
+func (cs ConsoleSlice) Print(output io.Writer) error {
```

Comment:
There's a potential regression here - the output doesn't seem to be aligned properly.

```console
$ go run cmd/theatre-consoles/main.go -c lab -n staging list
NAME                         NAMESPACE   PHASE     CREATED                         USER                         REASON
load-testing-default-7vpzf   staging     Stopped   2020-04-21 14:11:32 +0100 BST   benwheatley@gocardless.com
load-testing-default-n89cw   staging   Stopped   2020-04-21 14:08:32 +0100 BST   benwheatley@gocardless.com

$ $INTERNAL_CLI consoles list --context lab --namespace staging --labels app=load-testing
NAME                        PHASE    CREATED                        USER                        REASON
load-testing-default-7vpzf  Stopped  2020-04-21 14:11:32 +0100 BST  benwheatley@gocardless.com
load-testing-default-n89cw  Stopped  2020-04-21 14:08:32 +0100 BST  benwheatley@gocardless.com
```